src/platform/Tizen/WiFiManager.cpp

```diff
@@ -219,6 +344,89 @@ void WiFiManager::_ScanFinishedCb(wifi_manager_error_e wifiErr, void * userData)
     }
 }
 
+void WiFiManager::_ScanFinishedCb(wifi_manager_error_e wifiErr, void * userData)
+{
+    if (wifiErr == WIFI_MANAGER_ERROR_NONE)
+    {
+        ChipLogProgress(DeviceLayer, "WiFi scan finished");
+        std::vector<WiFiScanResponse> networkScanned;
+
+        int err = wifi_manager_foreach_found_specific_ap(sInstance.mWiFiManagerHandle, _FoundAPOnScanCb, &networkScanned);
+        if (err == WIFI_MANAGER_ERROR_NONE)
```

Comment:
In the line 368 you are logging `wifiErr`m not `err`, but since the value returned by the `wifi_manager_foreach_found_specific_ap` is in fact `wifi_manager_error_e` you can do:

```suggestion
        wifiErr = wifi_manager_foreach_found_specific_ap(sInstance.mWiFiManagerHandle, _FoundAPOnScanCb, &networkScanned);
        if (wifiErr == WIFI_MANAGER_ERROR_NONE)
```

However, also I would suggest to use VerifyOrReturn when checking for errors to improve the readability, so errors like the above will be easier to spot.